clang/test/Preprocessor/Inputs/file2.h.new

```diff
@@ -0,0 +1,11 @@
+#include "footer2.h"
```

Comment:
Was this file uploaded by mistake?

Reply:
This file is included in the `footer.h` file.

Reply:
Are you talking about ``footer2.h``?  I think @elizabethandrews is asking about ``file2.h.new``.

Reply:
Removed.